### ISSUE TITLE
MD041 (first-line-h1) and HTML comments

### ISSUE THREAD

**Issue by EvgenyOrekhov** [DavidAnson/markdownlint#37]
I find it convenient to disable some rules for the whole file by adding a markdownlint marker to the top of the file (as I would do with JSLint directives, for example):

```md
<!-- markdownlint-disable line-length -->
# Header

...
```

But this violates MD041 (first-line-h1) rule.
I think MD041 (first-line-h1) should ignore HTML comments.

**Comment by DavidAnson** [comment]
Agreed
